src/environment.ts

```diff
@@ -308,14 +308,18 @@ export class Environment {
       return isChrome();
     } else if (feature === 'WEBGL_CPU_FORWARD') {
       return true;
-    } else if (feature === 'WEBGL_PACK_BATCHNORMALIZATION') {
+    } else if (feature === 'WEBGL_PACK') {
       return false;
+    } else if (feature === 'WEBGL_PACK_BATCHNORMALIZATION') {
+      return this.get('WEBGL_PACK');
+    } else if (feature === 'WEBGL_PACK_CLIP') {
+      return this.get('WEBGL_PACK');
     } else if (feature === 'WEBGL_PACK_DEPTHWISECONV') {
-      return false;
+      return this.get('WEBGL_PACK');
     } else if (feature === 'WEBGL_LAZILY_UNPACK') {
-      return false;
+      return this.get('WEBGL_PACK');
```

Comment:
Minor issue: might be useful to note that this won't work if trying to turn on packing by calling set('WEBGL_PACK', true) from application code; it is still required to call set('WEBGL_LAZILY_UNPACK', true) since tf-layers initialization evaluates ENV.get('EPSILON') before application code, and causes evaluating and setting WEBGL_LAZILY_UNPACK to false.   
I'm aware that this should be done, properly, using URL query string parameters.  

Call stack:
Environment.evaluateFeature (environment.ts:299)
Environment.get (environment.ts:270)
MathBackendWebGL.compileAndRun (backend_webgl.ts:1988)
MathBackendWebGL.abs (backend_webgl.ts:1396)
(anonymous) (backend_webgl.ts:2029)
(anonymous) (engine.ts:156)
Engine.scopedRun (engine.ts:167)
Engine.tidy (engine.ts:153)
Environment.tidy (environment.ts:186)
MathBackendWebGL.floatPrecision (backend_webgl.ts:2028)
Environment.evaluateFeature (environment.ts:368)
Environment.get (environment.ts:270)
(anonymous) (common.ts:15)
extendStatics (tf-layers.js:20)
(anonymous) (tf-layers.js:21)

Reply:
Hey @astojilj - that's a good point - could you create a separate PR for this change?

Reply:
> Hey @astojilj - that's a good point - could you create a separate PR for this change?

I reported this so that you are aware of it when testing. Otherwise, I'm not sure what would be the desired behavior - setting master switch WEBGL_PACK would override all previously set sub-flags?